### ISSUE TITLE
Incompatibility with Carry-On mod.

### ISSUE THREAD

**Issue by Jaytche** [MehVahdJukaar/FastPaintings#8]
If you try to pick up a painting from this mod with the feature from the mod carry-on, you cannot ever put it back down, and you are forced to kill yourself to get rid of it.

**Comment by MehVahdJukaar** [comment]
that kind of seems a carry on issue. there could be many blocks in modded that are not meant to be picked up and could be not be forcefully placeable like that and if they dont probide with a way to drop them its definitely bad design from their end. Plus painting are a multiblock and should definitely be blacklisted so you can just do that

**Comment by Veynam** [comment]
Carry on does have a blacklist feature but sadly it doesnt work with this mod. Dont know if theres anything we can do about it

**Comment by MehVahdJukaar** [comment]
You sure you did that right? If blacklist is a tag are you sure you added the right id?

**Comment by Veynam** [comment]
Yes, I'm certain. Double checked too, unless I did something terribly wrong

**Comment by MehVahdJukaar** [comment]
Verify that with a mod that shows tags or stare at it with F3. If it's tagged report to carry on as it's their issue
